fe/fe-core/src/main/java/com/starrocks/common/FeConstants.java

```diff
@@ -71,6 +71,8 @@ public class FeConstants {
     public static int checkpoint_interval_second = 60; // 1 minutes
     // set to true to skip some step when running FE unit test
     public static boolean runningUnitTest = false;
+    // Set this flag to false to suppress showing local shuffle columns in verbose explain, when running FE unit tests.
+    public static boolean showLocalShuffleColumnsInExplain = true;
```

Comment:
Are session variables better?

Reply:
This is similar as `runningUnitTest`, just to avoid to modify too many test cases.

Reply:
can be always print it in `explain verbose/costs`?